model/labels/labels_slicelabels_test.go

```diff
@@ -23,3 +23,12 @@ var expectedSizeOfLabels = []uint64{ // Values must line up with testCaseLabels.
 	327,
 	549,
 }
+
+var expectedByteSize = []uint64{ // Values must line up with testCaseLabels.
+	72,
+	0,
+	97,
+	326,
+	327,
+	549,
```

Comment:
Not crucial, but I would probably have assigned this to `expectedSizeOfLabels` with a note that they are identical.